src/app/components/RawRoute.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { matchPath } from 'react-router-dom';
+
+const RawRoute = (routingConfig, location) => {
+  const routes = Object.keys(routingConfig.content).sort();
+  const nonRawPath = location.pathname.substring(4);
+
+  const route = routes.find(routeToMatch => matchPath(nonRawPath, routeToMatch));
+
+  if (route) {
+    const routeData = routingConfig.content[route].component.default;
+    const ComponentClass = routeData.componentClass;
+    const componentProps = Object.assign({}, routeData.props, { pathRoot: `/raw${route}` });
+    return <ComponentClass {...componentProps} />;
+  }
+  return 404;
```

Comment:
We need a terra crash page :D

Reply:
Agreed, but returning the text of 404 was cheaper for now. :)

Reply:
@neilpfeiffer wanna make us a sweet 404 page?

Reply:
Sounds good - I'll see what I can do.